### PR TITLE
Fix CDP error with pyenv version

### PR DESCRIPTION
**Pull request by rcillo**
CDP is logging

```
+ pip3 install -r requirements.txt
pyenv: version `3.6.9' is not installed (set by PYENV_VERSION environment variable)
Step finished unsuccessfully.
Step 'Run tests' finished unsuccessfully.
```

Probably the base image being openjdk we need to first install python
tooling before using it. I don't have a lot of experience with python so I'm not sure. Maybe it's just missing some env variable, so I'm trying out.

### CODE REVIEW

**Comment by antban** [comment]
:-1:

**Comment by antban** [comment]
The thing is that it's a message from cdp, related to incorrect overlay installation (delivery.yaml), not Dockerfile

**Comment by antban** [comment]
Just remove this line: https://github.com/zalando-nakadi/bubuku/blob/master/delivery.yaml#L7

**Comment by rcillo** [comment]
Ok, finally got it working. I had to change the version to `3.6.10` as `3.6.9` seems to no longer be available.

**Comment by ferbncode** [comment]
:+1:

File: delivery.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ pipeline:
     type: script
     overlay: guild-python/legacy
     env:
-        PYENV_VERSION: 3.6.9
+        PYENV_VERSION: 3.6.10
```
Comment (adyach) [inline delivery.yaml:7]: line https://github.com/zalando-nakadi/bubuku/pull/148/files#diff-8f20c8ae917a08b05bc008d227a1ad10R28 should it be `bubuku-appliance` vs `bubuku` ?

**Comment by rcillo** [comment]
👍

File: delivery.yaml (hunk 1 above)
Comment (adyach) [reply delivery.yaml:7]: line 28

Comment (rcillo) [reply delivery.yaml:7]: @adyach probably. To be honest I didn't understand why it publishes to identical images. Is it the case that "test" means for validation on PRs and the other is after merging?

Comment (rcillo) [reply delivery.yaml:7]: ef96170

Comment (adyach) [reply delivery.yaml:7]: i do not know the reason, usually, the image is the same, but configuration is different

**Comment by adyach** [comment]
👍

**Comment by rcillo** [comment]
👍

**Pull request merged**